### DISCUSSION TITLE
Can't figure out how to setup SSL with DockerCompose install

### DISCUSSION THREAD

**Discussion by sTiKyt** [firefly-iii/firefly-iii#6629] [Firefly III]
I've installed firefly III through Docker Compose, it works great but only with HTTP, different mobile apps require you to use HTTPS, so I was trying to figure out how to set up SSL, but there is no beginner-friendly guide for my case

**Comment by JC5** [comment]
You'll have to setup a reverse proxy. That's something you could Google maybe?

**Reply by sTiKyt** [reply]
Will look into it when I'm back home

**Comment by Drew-Goddyn** [comment]
Check out [ngnix reverse proxy ](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)
